ChiModules/LinearBoltzmannSolver/Groupset/lbs_groupset.cc

```diff
@@ -11,34 +11,9 @@
 
 //##############################################
 /**Groupset constructor.*/
-lbs::LBSGroupset::LBSGroupset(int in_id) : id(in_id)
+lbs::LBSGroupset::LBSGroupset(int in_id) :
+  id(in_id)
 {
```

Comment:
Here I moved all these stupid constructor assignments to in-line variants in the header file.